### DISCUSSION TITLE
Can we be work frens?

### DISCUSSION THREAD

**Discussion by yosStudent** [community/community#115952] [GitHub Education]
### Hi everyone,

Hi guys!!! I'm looking forward to someone who can help me get an associate or intern position :). I can work online or locally in Szczecin, Poland. I would prefer something C++ or Python related, also interested in robotics, but would be grateful for any of the suggestions offered.

P.S. I would also be very grateful if you could give me some tips on how to find my first job/internship.

I believe in a strong GitHub community!

Translated with DeepL.com (free version)

**Comment by MarouaneBenbetka** [comment]
Check Linkedin , make a good profile with all your experiences and then search for open jobs/internships , you can find good opprtunities.

**Reply by yosStudent** [reply]
Nice way, already done, but I'm currently triyng to find someone who will refer me ^_^
